Apps/W1/File - Azure BLOB Storage Connector/app/src/BlobStorageConnectorImpl.Codeunit.al

```diff
@@ -0,0 +1,414 @@
+
+    /// <summary>
+    /// Gets a List of Files stored on the provided account.
+    /// </summary>
+    /// <param name="AccountId">The file account ID which is used to get the file.</param>
+    /// <param name="Path">The file path to list.</param>
+    /// <param name="Files">A list with all files stored in the path.</param>
+    procedure ListFiles(AccountId: Guid; Path: Text; var FileAccountContent: Record "File Account Content" temporary)
+    var
+        ABSContainerContent: Record "ABS Container Content";
+        ABSBlobClient: Codeunit "ABS Blob Client";
+        ABSOperationResponse: Codeunit "ABS Operation Response";
+        ABSOptionalParameters: Codeunit "ABS Optional Parameters";
+    begin
+        InitBlobClient(AccountId, ABSBlobClient);
+        if (Path <> '') and not Path.EndsWith('/') then
```

Comment:
This should be defintily placed in a procedure an reused on all necessary places.